includes/log-handlers/class-wc-log-handler-file.php

```diff
@@ -266,7 +266,7 @@ public function remove( $handle ) {
 	protected function should_rotate( $handle ) {
 		$file = self::get_log_file_path( $handle );
 		if ( $file ) {
-			if ( $this->is_open( $handle ) ) {
+			if ( $this->is_open( $handle ) && is_resource( $this->handles[ $handle ] ) ) {
```

Comment:
Do you think `is_resource` should be added to the `is_open` method? `is_open` could prune handles that aren't resources, it looks like most calls are `is_open && is_resource`.